pkgs/tools/misc/task-keeper/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, rustPlatform, fetchFromGitHub }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "task-keeper";
+  version = "0.13.1";
+
+  src = fetchFromGitHub {
+    owner = "tennox"; # Using my fork because of https://github.com/linux-china/task-keeper/pull/5
+    repo = "task-keeper";
+    rev = "2a793dd7082259eab084f62bfd3a281f93c82a74";
+    hash = "sha256-oPdg8abAM+uSzx0OhRjDlTf4GdfmT9ALqTaH8u/9PUk=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/Cargo.lock";
```

Comment:
the lockfile has to be copied to nixpkgs, unfortunately

Reply:
I think it's because `lockFile` can't be evaluated without downloading the source. This sort of trickery isn't allowed in nixpkgs. See https://github.com/NixOS/nixpkgs/tree/master/pkgs#import-from-derivation

Reply:
Alright, thanks for the explanation - I've [committed](https://github.com/NixOS/nixpkgs/pull/236516/commits/aef94e6b6b8cc62510cbaa9445b16850c9c9d051) to nixpkgs & set `lockFile = "${./.}/Cargo.lock";`.
This works for me now:
`nix-build --option allow-import-from-derivation false -A task-keeper`

I've also updated to most recent version & bumped my PR to the upstream repo to commit Cargo.lock to there.